src/native/crt.c

```diff
@@ -320,6 +320,27 @@ int aws_size_t_from_java(JNIEnv *env, size_t *out_size, jlong java_long, const c
     return AWS_OP_ERR;
 }
 
+int aws_uint64_t_from_java_Long(JNIEnv *env, uint64_t *out_long, jobject java_Long, const char *errmsg_prefix) {
+    jlong jlong_value = (*env)->CallLongMethod(env, java_Long, boxed_long_properties.long_value_method_id);
```

Comment:
since you're making this a helper function, might be worth a null-check?
```suggestion
    if (java_long == NULL) {
        aws_jni_throw_null_pointer_exception(env, "%s can't be null");
        goto error;
    }

    jlong jlong_value = (*env)->CallLongMethod(env, java_Long, boxed_long_properties.long_value_method_id);
```